jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -446,7 +447,25 @@ public boolean isSessionRenewedOnAuthentication()
      */
     public void setSessionRenewedOnAuthentication(boolean renew)
     {
-        _renewSession = renew;
+        _renewSessionOnAuthentication = renew;
+    }
+
+    @Override
+    public int getSessionMaxInactiveIntervalOnAuthentication()
+    {
+        return _sessionMaxInactiveIntervalOnAuthentication;
+    }
+
+    /**
+     * Set the interval in seconds to set with {@link javax.servlet.http.HttpSession#setMaxInactiveInterval(int)}
+     * when a session is newly authenticated.
+     * @param seconds An interval in seconds, which if greater than zero will be set as the
+     *        {@link javax.servlet.http.HttpSession#setMaxInactiveInterval(int) maxInactiveInterval}
+     *        on newly authenticated sessions.
+     */
+    public void setSessionMaxInactiveIntervalOnAuthentication(int seconds)
```

Comment:
```suggestion
    public void setAuthenticatedSessionMaxInactiveInterval(int seconds)
```

Also beware  that this is in seconds, but HttpSession.setMaxInactiveInterval is  expecting minutes.  The javadoc on this method is misleading in this regard I think.

Reply:
Javadoc says interval is in seconds for the method we are calling.